src/alks.ts

```diff
@@ -204,6 +207,16 @@ namespace ALKS {
     machineIdentityArn: string;
   }
 
+  /**
+   * Alks role tags
+   * @property tagKey - The AWS role tag key
```

Comment:
Same here, seems like convention is for the @property to have the same name as the actual property in the object